lib/megaphone_client/episode.rb

```diff
@@ -0,0 +1,72 @@
+require "ostruct"
+require "json"
+require "rest-client"
+
+module MegaphoneClient
+
+  # @author Jay Arella
+  class Episode
+    class << self
+
+      # @return a MegaphoneClient
+      # @note This is used as a way to access top level attributes
+      # @example Accessing a network id
+      #   config.network_id #=> '{network id specified in initialization}'
+
+      def config
+        @config ||= MegaphoneClient
+      end
+
+      # @return a struct (or array of structs) that represents the search results by episode
+      # @note If an organizationId wasn't given in options and there is an organization_id in config,
+      #   it merges the organization_id from config into the params object that will be passed into MegaphoneClient#connection
+      # @see MegaphoneClient#connection
+      # @example Search for an episode with externalId 'show_episode-12345'
+      #   @megaphone.episode.search({
+      #     externalId: 'show_episode-1245'
+      #   })
+      #
+      #   #=> A struct representing 'show_episode-12345'
+
+      def search options={}
+        params = options
```

Comment:
It's a little unclear why we instantiate a new variable `params` here. Is there a reason why we want to preserve `options` even though we're merging stuff as well?

Reply:
Ah! Good point! I think that may have been an implementation detail that I forgot to remove as the it evolved. Probably makes more sense to just name it params from the beginning

Reply:
I think it's in case the following if-statement evaluates to false, but it could be replaced by using `#merge!`:

```ruby
        # If an organizationId wasn't given in options and there is an organization_id in config
        if !options[:organizationId] && config.organization_id
          # Merge the organization_id from config into the options object
          options.merge!({ organizationId: config.organization_id })
        end
```

Reply:
Ah! Thanks @Ravenstine!  I'm going to use this